main.md

```diff
@@ -536,15 +536,15 @@ that discloses only `region` and `country` of an `address` property:
 {#example-simple_structured-release-payload}
 ```json
 {
-  "nonce": "2GLC42sKQveCfGfryNRN9w",
-  "aud": "https://example.com/verifier",
+  "nonce": "s7bUT6edl1ystnrZmLiyWA",
+  "aud": "https://example.com/issuer",
   "_sd": {
-    "given_name": "[\"6Ij7tM-a5iVPGboS5tmvVA\", \"John\"]",
-    "family_name": "[\"eI8ZWm9QnKPpNPeNenHdhQ\", \"Doe\"]",
-    "birthdate": "[\"5bPs1IquZNa0hkaFzzzZNw\", \"1940-01-01\"]",
+    "given_name": "[\"eluV5Og3gSNII8EYnsxA_A\", \"John\"]",
+    "family_name": "[\"6Ij7tM-a5iVPGboS5tmvVA\", \"Doe\"]",
```

Comment:
@danielfett probably in the update example we may use the same salt to prevent useless changes in the draft

Reply:
Ok, I'm doing it